pkg/apk/implementation.go

```diff
@@ -375,7 +375,7 @@ func (a *APK) InitKeyring(ctx context.Context, keyFiles, extraKeyFiles []string)
 
 			var asURL *url.URL
 			var err error
-			if strings.HasPrefix(element, "https://") {
+			if strings.HasPrefix(element, "https://") || strings.HasPrefix(element, "s3://") {
```

Comment:
WDYT about https://gocloud.dev/howto/blob/ instead here? This could give us a standard interface for non-S3 APIs. AFAIK GCS's S3 compatibility is "best effort" which means not great.

Reply:
I think that would be fine to swap to if you'd prefer, though as I understand it pretty all blob storage implement the basic S3 APIs with full compatibility, it's as soon as you start using the more complex features like ACLs, CORS, etc that you start to run into issues. Given this is only ever calling `GetObject` I wouldn't expect to run into those issues.